app/src/androidTest/java/com/google/samples/apps/topeka/activity/SignInActivityTest.java

```diff
@@ -65,20 +72,142 @@ public void checkFab_initiallyNotDisplayed() {
     }
 
     @Test
+    public void signIn_checkIfFirstNameEditTextIsInitiallyEmpty() {
+        onView(withId(R.id.first_name)).check(matches(withText(isEmptyOrNullString())));
+    }
+
+    @Test
+    public void signIn_checkIfLastInitialEditTextIsInitiallyEmpty() {
+        onView(withId(last_initial)).check(matches(withText(isEmptyOrNullString())));
+    }
+
+    @Test
     public void signIn_performSuccessful() {
         inputData();
-        onView(withId(R.id.done)).check(matches(isDisplayed()));
+        onView(withId(R.id.done)).check(matches(isDisplayed())).perform(click());
+    }
+
+    @Test
+    public void signIn_chooseAvatar_01() {
```

Comment:
I really don't get it I used in a few commits before "for" loop and you said that my test checks to much. Now I see you made the same with avatars.